src/simba/review_daily_collections/includes/header.html

```
@@ -0,0 +1,11 @@
+<div class="flex flex-wrap justify-between items-center gap-2">
+  <div>
+    <h1 class="text-lg md:text-xl font-semibold text-stone-800 dark:text-neutral-200">
+      Review Daily Collections ({{daily_collections|length}})
+    </h1>
+    <p class="text-sm text-stone-500 dark:text-neutral-400">Review and verify daily collections or flag issues for follow-up.</p>
+    </p>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </p> ]

(tag-pair)